### DISCUSSION TITLE
Managing legacy file system patient data and permissions in modern data science stacks

### DISCUSSION THREAD

**Discussion by epijim** [rinpharma/positconf-roundtables-2023#4] [Ideas]
Historically XPT and sas7bdat were used in clinical reporting, with data saved to filesystems/network shares. 

This adds complexity with the growth in IDDs (like shiny apps), where filesystem permissions like acls are common. New technologies like parquet + s3 + athena also allow the mixing of mounted filesystem access, and portable API based access to co-exist. 

What experiences have companies had to date? What experiments / PoCs are under way? And where are the gaps we could collectively tackle?

**Comment by dgkf-roche** [comment]
Glad this is getting raised. Specifically, I'm interested to hear how companies would manage the transition from using data-from-a-database internally to submitting code that operates on data-on-the-filesystem. 

Are we aiming for all submission code to run end-to-end? How does that constrain the source of data for submission analysis? I haven't heard much from the Accumulus folks recently, but I think they had some ideas of cloud-accessible data for submissions.

**Comment by Coutuan3s** [comment]
Considering the premise of this topic on newer data tech, are we all aligned on this project?
CDISC and PHUSE are delighted to announce a new pilot project aimed at supporting the adoption of Dataset-JSON as an alternative transport format for regulatory submissions. 
Since we don't change our submission process too often are we sure this will support our Omics and wearable needs? Should we really keep prehistoric var name conventions? I tried to find documentation that would provide analysis of different options with pros/cons but maybe I did not look in the right places. I would be happy to get insights from member of this group.
